src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -370,8 +370,8 @@ public void runTests(boolean waitForApplicationUpdate, int messageOccurrences, T
                     long timeout = appStartupTimeout * 1000;
 
                     // Wait for the app started message in messages.log
-                    info("Waiting up to " + appStartupTimeout + " seconds for the application to start up...");
-                    String startMessage = serverTask.waitForStringInLog(START_APP_MESSAGE_REGEXP, timeout,
+                    info("Waiting up to " + appStartupTimeout + " seconds to find the application start up or update message...");
```

Comment:
The message should be worded to be more user-oriented e.g. `for the application to start up or update...`